py_configuration_files/course_discovery.py

```diff
@@ -18,7 +18,7 @@
     'http://localhost:1991',  # frontend-app-admin-portal
     'http://localhost:18400',  # frontend-app-publisher
     'http://localhost:18450',  # frontend-app-support-tools
-    'http://localhost:2000',  # frontend-app-learning
+    'http://localhost:2010',  # frontend-app-learning (port changed from 2000 to 2010, due to conflict in codepsace)
```

Comment:
See earlier comment about whether or not we might drop all these redundant comment changes.